etna/whatson/views.py

```diff
@@ -0,0 +1,18 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+
+@csrf_exempt
+def eventbrite_webhook_view(request):
+    if request.method == "POST":
+        # Parse the JSON data from the request
+        # json_data = json.loads(request.body)
+
```

Comment:
At this stage there are alot of unknowns, like what we receive, how much control we have over the granularity of the webhooks and whether we send an acknowledgement to EB. As a starting point, though, this looks fine.